<inh f='boost-1.85.0/libs/coroutine/include/boost/coroutine/exceptions.hpp' l='72' c='boost::coroutines::coroutine_error'/>
<def f='boost-1.85.0/libs/coroutine/include/boost/coroutine/exceptions.hpp' l='87' ll='95'/>
<size>40</size>
<fun r='_ZN5boost10coroutines14invalid_resultC1Ev'/>
